pkg/beacon/relay/dkg/result/submission.go

```diff
@@ -61,11 +61,15 @@ func (sm *SubmittingMember) SubmitDKGResult(
 		)
 	}
 
-	if len(signatures) < config.HonestThreshold {
+	// Chain rejects the result if it has less than 25% safety margin.
+	// If there are not enough signatures to preserve the margin, it does not
```

Comment:
Maybe we can also mention in the comment that we'd need total of 75% signatures?

Reply:
Technically, from the client's perspective, it doesn't have to be 75% if the on-chain threshold parameter is different from what we currently have. That's also why do we need to re-calculate this parameter here.